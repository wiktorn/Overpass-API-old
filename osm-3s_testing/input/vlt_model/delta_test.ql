[diff:"2018-02-01T00:01:00Z","2018-02-01T01:01:00Z"];

(
  make point ::geom=pt(48.1,11.004);
  make point ::geom=pt(48.1,11.005);
  make point ::geom=pt(48.1,11.006);
)->.point;

node(2001)->.n;
.n convert equal_geom ::=::;
out geom;
.n convert equal_geom ::=::,::geom=pt(48,11);
out geom;
.n convert equal_geom ::=::,::geom=lstr(pt(48,10.999),pt(48.001,11),pt(48,11.001));
out geom;
.n convert equal_geom ::=::,::geom=poly(lstr(pt(48,10.999),pt(48.001,11),pt(48,11.001)));
out geom;
.n convert equal_geom ::=::,::geom=point.gcat(geom());
out geom;
compare()(
  make marker after=n.set(id());out;
);

node(2003)->.n;
.n convert equal_geom ::=::;
out geom;
.n convert equal_geom ::=::,::geom=pt(48,11);
out geom;
.n convert equal_geom ::=::,::geom=lstr(pt(48,10.999),pt(48.001,11),pt(48,11.001));
out geom;
.n convert equal_geom ::=::,::geom=poly(lstr(pt(48,10.999),pt(48.001,11),pt(48,11.001)));
out geom;
.n convert equal_geom ::=::,::geom=point.gcat(geom());
out geom;
compare()(
  make marker after=n.set(id());out;
);

node(2004)->.n;
.n convert equal_geom ::=::;
out geom;
.n convert equal_geom ::=::,::geom=pt(48,11);
out geom;
.n convert equal_geom ::=::,::geom=lstr(pt(48,10.999),pt(48.001,11),pt(48,11.001));
out geom;
.n convert equal_geom ::=::,::geom=poly(lstr(pt(48,10.999),pt(48.001,11),pt(48,11.001)));
out geom;
.n convert equal_geom ::=::,::geom=point.gcat(geom());
out geom;
compare()(
  make marker after=n.set(id());out;
);

node(2005)->.n;
.n convert equal_geom ::=::;
out geom;
.n convert equal_geom ::=::,::geom=pt(48,11);
out geom;
.n convert equal_geom ::=::,::geom=lstr(pt(48,10.999),pt(48.001,11),pt(48,11.001));
out geom;
.n convert equal_geom ::=::,::geom=poly(lstr(pt(48,10.999),pt(48.001,11),pt(48,11.001)));
out geom;
.n convert equal_geom ::=::,::geom=point.gcat(geom());
out geom;
compare()(
  make marker after=n.set(id());out;
);

(
  make point ::geom=pt(48.1,11.004);
  make point ::geom=pt(48.1,11.0055);
  make point ::geom=pt(48.1,11.006);
)->.point_bis;

node(2500)->.n;
if (n.count(nodes) == 1)
(
  make equal_tags tag="no_geom"->.no_geom;
  make equal_tags tag="center",::geom=pt(48,12.004)->.center;
  make equal_tags tag="comp_vs_vertex",::geom=point.gcat(geom())->.comp_vs_vertex;
  make equal_tags tag="comp_equal",::geom=point.gcat(geom())->.comp_equal;
  make equal_tags tag="comp_vertex_diff",::geom=point.gcat(geom())->.comp_vertex_diff;
  make equal_tags tag="link_equal",::geom=lstr(pt(48,10.999),pt(48.001,11),pt(48,11.001))->.link_equal;
  make equal_tags tag="link_last_diff",::geom=lstr(pt(48,10.999),pt(48.001,11),pt(48.0005,10.9999))->.link_last_diff;
  make equal_tags tag="link_first_diff",::geom=lstr(pt(48.0005,11.0001),pt(48.001,11),pt(48,11.001))->.link_first_diff;
  make equal_tags tag="link_middle_diff",::geom=lstr(pt(48,10.999),pt(48.001,11),pt(48,11.001))->.link_middle_diff;
  make equal_tags tag="poly_equal_1",::geom=poly(lstr(pt(48,12),pt(48,12.001),pt(48.001,12.001)))->.poly_equal_1;
  make equal_tags tag="poly_equal_2",::geom=poly(
      lstr(pt(48,12),pt(48,12.001),pt(48.001,12.001)),
      lstr(pt(48,13),pt(48,13.001),pt(48.001,13.001)))->.poly_equal_2;
  make equal_tags tag="poly_diff_1",::geom=poly(
      lstr(pt(48,12),pt(48,12.001),pt(48.001,12.001)),
      lstr(pt(48,13),pt(48,13.001),pt(48.001,13.001)))->.poly_diff_1;
  make equal_tags tag="poly_diff_2",::geom=poly(
      lstr(pt(48,12),pt(48,12.001),pt(48.001,12.001)),
      lstr(pt(48,13),pt(48,13.001),pt(48.001,13.001)))->.poly_diff_2;
);
if (n.count(nodes) == 0)
(
  make equal_tags tag="no_geom",::geom=pt(48,12)->.no_geom;
  make equal_tags tag="center",::geom=pt(48,12.006)->.center;
  make equal_tags tag="comp_vs_vertex",::geom=center(point.gcat(geom()))->.comp_vs_vertex;
  make equal_tags tag="comp_equal",::geom=point.gcat(geom())->.comp_equal;
  make equal_tags tag="comp_vertex_diff",::geom=point_bis.gcat(geom())->.comp_vertex_diff;
  make equal_tags tag="link_equal",::geom=lstr(pt(48,10.999),pt(48.001,11),pt(48,11.001))->.link_equal;
  make equal_tags tag="link_last_diff",::geom=lstr(pt(48,10.999),pt(48.001,11),pt(48.0006,10.9999))->.link_last_diff;
  make equal_tags tag="link_first_diff",::geom=lstr(pt(48.0006,11.0001),pt(48.001,11),pt(48,11.001))->.link_first_diff;
  make equal_tags tag="link_middle_diff",::geom=lstr(pt(48,10.999),pt(48.001,11.0001),pt(48,11.001))->.link_middle_diff;
  make equal_tags tag="poly_equal_1",::geom=poly(lstr(pt(48,12),pt(48,12.001),pt(48.001,12.001)))->.poly_equal_1;
  make equal_tags tag="poly_equal_2",::geom=poly(
      lstr(pt(48,12),pt(48,12.001),pt(48.001,12.001)),
      lstr(pt(48,13),pt(48,13.001),pt(48.001,13.001)))->.poly_equal_2;
  make equal_tags tag="poly_diff_1",::geom=poly(
      lstr(pt(48,12),pt(48.0001,12.001),pt(48.001,12.001)),
      lstr(pt(48,13),pt(48,13.001),pt(48.001,13.001)))->.poly_diff_1;
  make equal_tags tag="poly_diff_2",::geom=poly(
      lstr(pt(48,12),pt(48,12.001),pt(48.001,12.001)),
      lstr(pt(48,13),pt(48.0001,13.001),pt(48.001,13.001)))->.poly_diff_2;
);
.no_geom out geom;
.center out geom;
.comp_vs_vertex out geom;
.comp_equal out geom;
.comp_vertex_diff out geom;
.link_equal out geom;
.link_last_diff out geom;
.link_first_diff out geom;
.link_middle_diff out geom;
.poly_equal_1 out geom;
.poly_equal_2 out geom;
.poly_diff_1 out geom;
.poly_diff_2 out geom;
