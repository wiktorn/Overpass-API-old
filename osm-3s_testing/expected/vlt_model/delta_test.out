<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="Overpass API">
<note>The data included in this document is from www.openstreetmap.org. The data is made available under ODbL.</note>
<meta osm_base=""/>

<action type="delete">
<old>
  <equal_geom id="4">
    <tag k="changes" v="old_value"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="43">
    <tag k="changes" v="new_value"/>
  </equal_geom>
</action>
<action type="delete">
<old>
  <equal_geom id="5">
    <point lat="48.0000000" lon="11.0000000"/>
    <tag k="changes" v="old_value"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="44">
    <point lat="48.0000000" lon="11.0000000"/>
    <tag k="changes" v="new_value"/>
  </equal_geom>
</action>
<action type="delete">
<old>
  <equal_geom id="6">
    <vertex lat="48.0000000" lon="10.9990000"/>
    <vertex lat="48.0010000" lon="11.0000000"/>
    <vertex lat="48.0000000" lon="11.0010000"/>
    <tag k="changes" v="old_value"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="45">
    <vertex lat="48.0000000" lon="10.9990000"/>
    <vertex lat="48.0010000" lon="11.0000000"/>
    <vertex lat="48.0000000" lon="11.0010000"/>
    <tag k="changes" v="new_value"/>
  </equal_geom>
</action>
<action type="delete">
<old>
  <equal_geom id="7">
    <linestring>
      <vertex lat="48.0000000" lon="10.9990000"/>
      <vertex lat="48.0000000" lon="11.0010000"/>
      <vertex lat="48.0010000" lon="11.0000000"/>
      <vertex lat="48.0000000" lon="10.9990000"/>
    </linestring>
    <tag k="changes" v="old_value"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="46">
    <linestring>
      <vertex lat="48.0000000" lon="10.9990000"/>
      <vertex lat="48.0000000" lon="11.0010000"/>
      <vertex lat="48.0010000" lon="11.0000000"/>
      <vertex lat="48.0000000" lon="10.9990000"/>
    </linestring>
    <tag k="changes" v="new_value"/>
  </equal_geom>
</action>
<action type="delete">
<old>
  <equal_geom id="8">
    <group>
      <point lat="48.1000000" lon="11.0040000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0050000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0060000"/>
    </group>
    <tag k="changes" v="old_value"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="47">
    <group>
      <point lat="48.1000000" lon="11.0040000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0050000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0060000"/>
    </group>
    <tag k="changes" v="new_value"/>
  </equal_geom>
</action>
<action type="show_initial">
  <marker id="48">
    <tag k="after" v="2001"/>
  </marker>
</action>
<action type="show_final">
  <marker id="49">
    <tag k="after" v="2001"/>
  </marker>
</action>
<action type="show_initial">
  <marker id="55">
    <tag k="after" v="2003"/>
  </marker>
</action>
<action type="show_final">
  <marker id="56">
    <tag k="after" v="2003"/>
  </marker>
</action>
<action type="delete">
<old>
  <equal_geom id="14">
    <tag k="one" v="const_2004"/>
    <tag k="two" v="const_2004"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="57">
    <tag k="one" v="const_2004"/>
    <tag k="three" v="now_2004"/>
    <tag k="two" v="const_2004"/>
  </equal_geom>
</action>
<action type="delete">
<old>
  <equal_geom id="15">
    <point lat="48.0000000" lon="11.0000000"/>
    <tag k="one" v="const_2004"/>
    <tag k="two" v="const_2004"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="58">
    <point lat="48.0000000" lon="11.0000000"/>
    <tag k="one" v="const_2004"/>
    <tag k="three" v="now_2004"/>
    <tag k="two" v="const_2004"/>
  </equal_geom>
</action>
<action type="delete">
<old>
  <equal_geom id="16">
    <vertex lat="48.0000000" lon="10.9990000"/>
    <vertex lat="48.0010000" lon="11.0000000"/>
    <vertex lat="48.0000000" lon="11.0010000"/>
    <tag k="one" v="const_2004"/>
    <tag k="two" v="const_2004"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="59">
    <vertex lat="48.0000000" lon="10.9990000"/>
    <vertex lat="48.0010000" lon="11.0000000"/>
    <vertex lat="48.0000000" lon="11.0010000"/>
    <tag k="one" v="const_2004"/>
    <tag k="three" v="now_2004"/>
    <tag k="two" v="const_2004"/>
  </equal_geom>
</action>
<action type="delete">
<old>
  <equal_geom id="17">
    <linestring>
      <vertex lat="48.0000000" lon="10.9990000"/>
      <vertex lat="48.0000000" lon="11.0010000"/>
      <vertex lat="48.0010000" lon="11.0000000"/>
      <vertex lat="48.0000000" lon="10.9990000"/>
    </linestring>
    <tag k="one" v="const_2004"/>
    <tag k="two" v="const_2004"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="60">
    <linestring>
      <vertex lat="48.0000000" lon="10.9990000"/>
      <vertex lat="48.0000000" lon="11.0010000"/>
      <vertex lat="48.0010000" lon="11.0000000"/>
      <vertex lat="48.0000000" lon="10.9990000"/>
    </linestring>
    <tag k="one" v="const_2004"/>
    <tag k="three" v="now_2004"/>
    <tag k="two" v="const_2004"/>
  </equal_geom>
</action>
<action type="delete">
<old>
  <equal_geom id="18">
    <group>
      <point lat="48.1000000" lon="11.0040000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0050000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0060000"/>
    </group>
    <tag k="one" v="const_2004"/>
    <tag k="two" v="const_2004"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="61">
    <group>
      <point lat="48.1000000" lon="11.0040000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0050000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0060000"/>
    </group>
    <tag k="one" v="const_2004"/>
    <tag k="three" v="now_2004"/>
    <tag k="two" v="const_2004"/>
  </equal_geom>
</action>
<action type="show_initial">
  <marker id="62">
    <tag k="after" v="2004"/>
  </marker>
</action>
<action type="show_final">
  <marker id="63">
    <tag k="after" v="2004"/>
  </marker>
</action>
<action type="delete">
<old>
  <equal_geom id="19">
    <tag k="one" v="const_2005"/>
    <tag k="three" v="old_value"/>
    <tag k="two" v="const_2005"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="64">
    <tag k="one" v="const_2005"/>
    <tag k="three" v="new_value"/>
    <tag k="two" v="const_2005"/>
  </equal_geom>
</action>
<action type="delete">
<old>
  <equal_geom id="20">
    <point lat="48.0000000" lon="11.0000000"/>
    <tag k="one" v="const_2005"/>
    <tag k="three" v="old_value"/>
    <tag k="two" v="const_2005"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="65">
    <point lat="48.0000000" lon="11.0000000"/>
    <tag k="one" v="const_2005"/>
    <tag k="three" v="new_value"/>
    <tag k="two" v="const_2005"/>
  </equal_geom>
</action>
<action type="delete">
<old>
  <equal_geom id="21">
    <vertex lat="48.0000000" lon="10.9990000"/>
    <vertex lat="48.0010000" lon="11.0000000"/>
    <vertex lat="48.0000000" lon="11.0010000"/>
    <tag k="one" v="const_2005"/>
    <tag k="three" v="old_value"/>
    <tag k="two" v="const_2005"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="66">
    <vertex lat="48.0000000" lon="10.9990000"/>
    <vertex lat="48.0010000" lon="11.0000000"/>
    <vertex lat="48.0000000" lon="11.0010000"/>
    <tag k="one" v="const_2005"/>
    <tag k="three" v="new_value"/>
    <tag k="two" v="const_2005"/>
  </equal_geom>
</action>
<action type="delete">
<old>
  <equal_geom id="22">
    <linestring>
      <vertex lat="48.0000000" lon="10.9990000"/>
      <vertex lat="48.0000000" lon="11.0010000"/>
      <vertex lat="48.0010000" lon="11.0000000"/>
      <vertex lat="48.0000000" lon="10.9990000"/>
    </linestring>
    <tag k="one" v="const_2005"/>
    <tag k="three" v="old_value"/>
    <tag k="two" v="const_2005"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="67">
    <linestring>
      <vertex lat="48.0000000" lon="10.9990000"/>
      <vertex lat="48.0000000" lon="11.0010000"/>
      <vertex lat="48.0010000" lon="11.0000000"/>
      <vertex lat="48.0000000" lon="10.9990000"/>
    </linestring>
    <tag k="one" v="const_2005"/>
    <tag k="three" v="new_value"/>
    <tag k="two" v="const_2005"/>
  </equal_geom>
</action>
<action type="delete">
<old>
  <equal_geom id="23">
    <group>
      <point lat="48.1000000" lon="11.0040000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0050000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0060000"/>
    </group>
    <tag k="one" v="const_2005"/>
    <tag k="three" v="old_value"/>
    <tag k="two" v="const_2005"/>
  </equal_geom>
</old>
</action>
<action type="create">
  <equal_geom id="68">
    <group>
      <point lat="48.1000000" lon="11.0040000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0050000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0060000"/>
    </group>
    <tag k="one" v="const_2005"/>
    <tag k="three" v="new_value"/>
    <tag k="two" v="const_2005"/>
  </equal_geom>
</action>
<action type="show_initial">
  <marker id="69">
    <tag k="after" v="2005"/>
  </marker>
</action>
<action type="show_final">
  <marker id="70">
    <tag k="after" v="2005"/>
  </marker>
</action>
<action type="delete">
<old>
  <equal_tags id="27">
    <tag k="tag" v="no_geom"/>
  </equal_tags>
</old>
</action>
<action type="create">
  <equal_tags id="74">
    <point lat="48.0000000" lon="12.0000000"/>
    <tag k="tag" v="no_geom"/>
  </equal_tags>
</action>
<action type="delete">
<old>
  <equal_tags id="28">
    <point lat="48.0000000" lon="12.0040000"/>
    <tag k="tag" v="center"/>
  </equal_tags>
</old>
</action>
<action type="create">
  <equal_tags id="75">
    <point lat="48.0000000" lon="12.0060000"/>
    <tag k="tag" v="center"/>
  </equal_tags>
</action>
<action type="delete">
<old>
  <equal_tags id="29">
    <group>
      <point lat="48.1000000" lon="11.0040000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0050000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0060000"/>
    </group>
    <tag k="tag" v="comp_vs_vertex"/>
  </equal_tags>
</old>
</action>
<action type="create">
  <equal_tags id="76">
    <point lat="48.1000000" lon="11.0050000"/>
    <tag k="tag" v="comp_vs_vertex"/>
  </equal_tags>
</action>
<action type="delete">
<old>
  <equal_tags id="31">
    <group>
      <point lat="48.1000000" lon="11.0040000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0050000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0060000"/>
    </group>
    <tag k="tag" v="comp_vertex_diff"/>
  </equal_tags>
</old>
</action>
<action type="create">
  <equal_tags id="78">
    <group>
      <point lat="48.1000000" lon="11.0040000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0055000"/>
    </group>
    <group>
      <point lat="48.1000000" lon="11.0060000"/>
    </group>
    <tag k="tag" v="comp_vertex_diff"/>
  </equal_tags>
</action>
<action type="delete">
<old>
  <equal_tags id="33">
    <vertex lat="48.0000000" lon="10.9990000"/>
    <vertex lat="48.0010000" lon="11.0000000"/>
    <vertex lat="48.0005000" lon="10.9999000"/>
    <tag k="tag" v="link_last_diff"/>
  </equal_tags>
</old>
</action>
<action type="create">
  <equal_tags id="80">
    <vertex lat="48.0000000" lon="10.9990000"/>
    <vertex lat="48.0010000" lon="11.0000000"/>
    <vertex lat="48.0006000" lon="10.9999000"/>
    <tag k="tag" v="link_last_diff"/>
  </equal_tags>
</action>
<action type="delete">
<old>
  <equal_tags id="34">
    <vertex lat="48.0005000" lon="11.0001000"/>
    <vertex lat="48.0010000" lon="11.0000000"/>
    <vertex lat="48.0000000" lon="11.0010000"/>
    <tag k="tag" v="link_first_diff"/>
  </equal_tags>
</old>
</action>
<action type="create">
  <equal_tags id="81">
    <vertex lat="48.0006000" lon="11.0001000"/>
    <vertex lat="48.0010000" lon="11.0000000"/>
    <vertex lat="48.0000000" lon="11.0010000"/>
    <tag k="tag" v="link_first_diff"/>
  </equal_tags>
</action>
<action type="delete">
<old>
  <equal_tags id="35">
    <vertex lat="48.0000000" lon="10.9990000"/>
    <vertex lat="48.0010000" lon="11.0000000"/>
    <vertex lat="48.0000000" lon="11.0010000"/>
    <tag k="tag" v="link_middle_diff"/>
  </equal_tags>
</old>
</action>
<action type="create">
  <equal_tags id="82">
    <vertex lat="48.0000000" lon="10.9990000"/>
    <vertex lat="48.0010000" lon="11.0001000"/>
    <vertex lat="48.0000000" lon="11.0010000"/>
    <tag k="tag" v="link_middle_diff"/>
  </equal_tags>
</action>
<action type="delete">
<old>
  <equal_tags id="38">
    <linestring>
      <vertex lat="48.0000000" lon="12.0000000"/>
      <vertex lat="48.0000000" lon="12.0010000"/>
      <vertex lat="48.0010000" lon="12.0010000"/>
      <vertex lat="48.0000000" lon="12.0000000"/>
    </linestring>
    <linestring>
      <vertex lat="48.0000000" lon="13.0000000"/>
      <vertex lat="48.0000000" lon="13.0010000"/>
      <vertex lat="48.0010000" lon="13.0010000"/>
      <vertex lat="48.0000000" lon="13.0000000"/>
    </linestring>
    <tag k="tag" v="poly_diff_1"/>
  </equal_tags>
</old>
</action>
<action type="create">
  <equal_tags id="85">
    <linestring>
      <vertex lat="48.0000000" lon="12.0000000"/>
      <vertex lat="48.0001000" lon="12.0010000"/>
      <vertex lat="48.0010000" lon="12.0010000"/>
      <vertex lat="48.0000000" lon="12.0000000"/>
    </linestring>
    <linestring>
      <vertex lat="48.0000000" lon="13.0000000"/>
      <vertex lat="48.0000000" lon="13.0010000"/>
      <vertex lat="48.0010000" lon="13.0010000"/>
      <vertex lat="48.0000000" lon="13.0000000"/>
    </linestring>
    <tag k="tag" v="poly_diff_1"/>
  </equal_tags>
</action>
<action type="delete">
<old>
  <equal_tags id="39">
    <linestring>
      <vertex lat="48.0000000" lon="12.0000000"/>
      <vertex lat="48.0000000" lon="12.0010000"/>
      <vertex lat="48.0010000" lon="12.0010000"/>
      <vertex lat="48.0000000" lon="12.0000000"/>
    </linestring>
    <linestring>
      <vertex lat="48.0000000" lon="13.0000000"/>
      <vertex lat="48.0000000" lon="13.0010000"/>
      <vertex lat="48.0010000" lon="13.0010000"/>
      <vertex lat="48.0000000" lon="13.0000000"/>
    </linestring>
    <tag k="tag" v="poly_diff_2"/>
  </equal_tags>
</old>
</action>
<action type="create">
  <equal_tags id="86">
    <linestring>
      <vertex lat="48.0000000" lon="12.0000000"/>
      <vertex lat="48.0000000" lon="12.0010000"/>
      <vertex lat="48.0010000" lon="12.0010000"/>
      <vertex lat="48.0000000" lon="12.0000000"/>
    </linestring>
    <linestring>
      <vertex lat="48.0000000" lon="13.0000000"/>
      <vertex lat="48.0001000" lon="13.0010000"/>
      <vertex lat="48.0010000" lon="13.0010000"/>
      <vertex lat="48.0000000" lon="13.0000000"/>
    </linestring>
    <tag k="tag" v="poly_diff_2"/>
  </equal_tags>
</action>

</osm>
